src/serac/physics/heat_transfer.hpp

```diff
@@ -24,52 +24,52 @@
 
 namespace serac {
 
-namespace Thermal {
+namespace heat_transfer {
 
 /**
  * @brief Reasonable defaults for most thermal linear solver options
- *
- * @return The default thermal linear options
  */
-IterativeSolverOptions defaultLinearOptions()
-{
-  return {.rel_tol     = 1.0e-6,
-          .abs_tol     = 1.0e-12,
-          .print_level = 0,
-          .max_iter    = 200,
-          .lin_solver  = LinearSolver::CG,
-          .prec        = HypreSmootherPrec{mfem::HypreSmoother::Jacobi}};
-}
+const IterativeSolverOptions default_linear_options = {.rel_tol     = 1.0e-6,
+                                                       .abs_tol     = 1.0e-12,
+                                                       .print_level = 0,
+                                                       .max_iter    = 200,
+                                                       .lin_solver  = LinearSolver::CG,
```

Comment:
The heat transfer operator will often be self-adjoint, so CG will *usually* be a good choice. However, this isn't guaranteed in general, and the linear solver could have asymmetric matrices to work with. Maybe we should stay on the safe side for defaults and use GMRES. What do you think?

Reply:
Sounds good to me. In fact, I think that's normally what I suggest to people. 😄